catalyst/registry/registry.py

```diff
@@ -142,12 +142,34 @@ def _schedulers_loader(r: Registry):
 SCHEDULERS.late_add(_schedulers_loader)
 Scheduler = SCHEDULERS.add
 
-
 EXPERIMENTS = Registry("experiment")
 Experiment = EXPERIMENTS.add
 
+RUNNERS = Registry("runner")
+Runner = RUNNERS.add
```

Comment:
I'd appreciate it if you would sort registers by importance (e.g. `EXPERIMENTS`, `Experiment`, ..., `RUNNERS`, `Runner`, ..., `_callbacks_loader`, `CALLBACKS`, `Callback`, ...) or alphabetically